### ISSUE TITLE
Sweeping Multisig Electrum Private keys into Electron cash to claim BCC

### ISSUE THREAD

**Issue by grajnikanth** [fyookball/electrum#351]
HI, I have BTC sitting in a multisig electrum wallet (2 of 2) for a while now. I bought them before the Bitcoin Cash Hardfork. I want to claim the BCC in a new electron cash wallet. But all the tutorials I see talk about non multisig electrum wallets and private keys. I created the mutlisig wallet as a safety measure with one wallet sitting on a air-gapped computer and with the second wallet is on a computer which is online. Where can I get a single private key which represents these wallets? So that I can sweep the private keys into the electron cash to claim my BCC.

**Comment by fyookball** [comment]
lets ask @kyuupichan as he has more experience with MS... i wouldnt be able to answer off the top of my head

**Comment by grajnikanth** [comment]
OK thanks, any help on this will be great. Will you ask @kyuupichan or will he check this post?

**Comment by kyuupichan** [comment]
Just set up your multisig wallet in electron cash the same way you did in electrum. The address will match and you can sign and send

**Comment by grajnikanth** [comment]
Ok great, I will try this out and let you all know how it worked.

**Comment by wpmonty** [comment]
@grajnikanth did this work? I'm having the same 2FA issue.

**Comment by grajnikanth** [comment]
Hi, I setup the electron cash wallet as suggested by @kyuupichan and it worked. Just to be safe I did this on two new computers. I used my office computer and an extra computer lying around the house. While setting it up, I used the option of creating the wallet using the seed for both the wallets. I used the two seeds which I got from the original electrum wallets. It worked fine and I see all my BCC. I have not done any transactions yet from this wallet. I am going to do that today. @wpmonty, let me know if you have any other questions on this.
